hash/pc8801_flop.xml

```diff
@@ -29118,29 +29121,30 @@ Start and Program Disks OK, the remaining part is damaged (missing d88 headers?)
 		</part>
 	</software>
 
-	<software name="sf3dopv">
-		<description>S.F.3.D Original Operation V</description>
-		<year>1985</year>
+	<software name="sf3dthxg">
+		<description>S.F.3.D Point X Senryou Sakusen - Original Operation Thanksgiving</description>
```

Comment:
> Box art of S.F.3.D Point X Senryou Sakusen - Operation Thanksgiving
... which I guess it's really expected to translate the kanji to English. May be worth adding a comment?
https://images.launchbox-app.com/1a057d08-4a1a-4cb0-a9f0-345082928c9c.jpg

Reply:
change alt_title hope it is ok now :)